zebra-consensus/src/config.rs

```diff
@@ -42,7 +78,15 @@ impl Default for Config {
     fn default() -> Self {
         Self {
             checkpoint_sync: true,
-            debug_skip_parameter_preload: false,
+        }
+    }
+}
+
+impl Default for InnerConfig {
+    fn default() -> Self {
+        Self {
+            checkpoint_sync: true,
```

Comment:
Let's avoid these defaults getting out of sync, particularly as we add more fields.

```suggestion
            checkpoint_sync: Config::default().checkpoint_sync,
```